### PR TITLE
eX buttons comply with Caseflow Commons styling

### PR DESCRIPTION
**Pull request by Sjones352**
Connects to https://github.com/department-of-veterans-affairs/caseflow-efolder/issues/565

## AC
1. Verify that all eX buttons are sentence case.
1. Verify that the height of all the eX buttons (including the new copy Vet ID button) follows Caseflow Commons.
1. Verify that the styling of the padding to the left and right of the button label is in accordance with Caseflow Commons.

screen shot 2017-08-03 at 12 07 19 pm

buttons

### CODE REVIEW

File: app/assets/stylesheets/_main.scss [hunk 1]
```diff
@@ -208,3 +208,12 @@ table td {
   color: $color-gray;
 }
 
+.cf-retrive-button.cf-submit {
```
Comment (mmlumba) [inline app/assets/stylesheets/_main.scss]: spelling nit: "retrieve"

File: app/assets/stylesheets/_main.scss [hunk 2]
```diff
@@ -208,3 +208,12 @@ table td {
...
+  padding-right: 20px;
+  padding-left: 20px;
+}
+
+.ee-copy-button {
```
Comment (mmlumba) [inline app/assets/stylesheets/_main.scss]: You can change this and the other class to the following:
`padding: 0 20px`
https://developer.mozilla.org/en-US/docs/Web/CSS/padding

**Comment by mmlumba** [comment]
@Sjones352 can you link to the original ticket?

File: test [hunk 3]
```diff
@@ -0,0 +1 @@
+plain text
```
Comment (mmlumba) [inline test]: This is a test file that runs after every test.  There's a pending ticket that will resolve to clear this file after every test run but for now delete it.

File: app/views/downloads/_confirm.html.erb [hunk 4]
```diff
@@ -90,11 +89,15 @@
 </div>
 
 <div class="cf-app-segment">
+<div class="cf-push-right">
```
Comment (mmlumba) [inline app/views/downloads/_confirm.html.erb]: The bottom part still looks like this (might be out of scope for this ticket):
screen shot 2017-08-03 at 2 14 29 pm

File: Gemfile [hunk 5]
```diff
@@ -1,5 +1,4 @@
 source ENV['GEM_SERVER_URL'] || 'https://rubygems.org'
-
```
Comment (Sjones352) [inline Gemfile]: Fixed

File: app/assets/stylesheets/_main.scss (hunk 1 above)
Comment (Sjones352) [inline app/assets/stylesheets/_main.scss]: Fixed

File: app/assets/stylesheets/_main.scss (hunk 2 above)
Comment (Sjones352) [inline app/assets/stylesheets/_main.scss]: Fixed

File: test (hunk 3 above)
Comment (Sjones352) [inline test]: Fixed

File: app/views/downloads/_confirm.html.erb (hunk 4 above)
Comment (Sjones352) [inline app/views/downloads/_confirm.html.erb]: Fixed

**Pull request merged**